CONTRIBUTING.md

```diff
@@ -11,3 +11,8 @@ be under the Apache 2 License, as dictated by that
    the terms of any separate license agreement you may have executed
    with Licensor regarding such Contributions.
 ~~~
+
+Contributors must sign-off each commit by adding a `Signed-off-by:` line
```

Comment:
```suggestion
Contributors must sign-off each commit by adding a `Signed-off-by: <author>` line
```

Reply:
The `<` `>` around `author` might be misleading since the actual line is:

> Signed-off-by: f\ull name <email>

Maybe just `Signed-off-by: ...` to indicate something is following? I don't want the paragraph to become documentation how to sign off commits :wink:

Reply:
Sure, that sounds good as well! I'm updating the suggestion in case you want to take it.

Reply:
At Bosch, we have been a notch more explicit about the form of the signature:
https://github.com/boschresearch/fmi_adapter/blob/master/CONTRIBUTING.md#sign-your-work

author can either be a full name or pseudonym, but an email address has to be provided as well.

Reply:
Ironic: I had to apply the suggestion manually - otherwise the DCO would fail for that commit :wink: